src/components/TodoHeader.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from 'react';
+import { Error, Todo } from '../types/Todo';
+import * as todosService from '../api/todos';
+
+type Props = {
+  todos: Todo[],
+  setTodos: (value: Todo[]) => void,
+  tempTodo: Todo | null,
+  setTempTodo: (value: null | Todo) => void,
+  setHasError: (value: Error) => void,
+  setLoading: (value: boolean) => void,
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  todos,
+  setTodos,
+  tempTodo,
+  setTempTodo,
+  setHasError,
+  setLoading,
+}) => {
```

Comment:
The same
![image](https://github.com/mate-academy/react_todo-app-add-and-delete/assets/54936044/e2c84cb6-de0b-46e3-bcda-5d4778d8bcbc)